model/task/task_test.go

```diff
@@ -1253,6 +1253,43 @@ func TestBulkInsert(t *testing.T) {
 	}
 }
 
+func TestGetTaskIdBetweenIds(t *testing.T) {
+	assert := assert.New(t)
+	require.NoError(t, db.ClearCollections(Collection))
+	displayName := "cool-task-9000"
+	buildVarient := "bv"
+	requester := "r"
+	project := "proj"
+	tasks := []Task{}
+	for i := 1; i <= 20; i++ {
+		task := Task{
+			Id:                  "t" + fmt.Sprint(i),
+			DisplayName:         displayName,
+			BuildVariant:        buildVarient,
+			Requester:           requester,
+			Project:             project,
+			RevisionOrderNumber: i,
+		}
+		assert.NoError(task.Insert())
+		tasks = append(tasks, task)
+	}
+	t10, err := GetTaskIdBetweenTasks(tasks[0], tasks[19])
+	assert.Equal(t10.RevisionOrderNumber, 10)
```

Comment:
Have to `require.NotZero(t10)` first or else this can panic when dereferencing `t10`. It's also a little more idiomatic to check `err` before checking the result.

Same with the others below.